test/rxd/testutils.py

```diff
@@ -126,3 +128,18 @@ def compare_data(data):
     )
     max_err = numpy.amax(abs(corr_vals.T - tst_dat[t2_0:t2_n, 1:]))
     return max_err
+
+
+def check_platform():
```

Comment:
The name is very generic. There's a few ways the name can change:

* `check_platform_multi_gridding_mix` if you expect each test to have it's on criteria.
* `check_platform_ubuntu24` if you think failure is likely more related to the OS than the test.
* `check_platform("ubuntu", "24")` then use `&&` and `!` to ignore exactly the problematic versions.
 
or whatever variation you feel looks nice.

Reply:
Thank you for clarifying the versions! Then maybe `check_platforms("ubuntu", versions=[22, 24])` is a convenient API?

Reply:
So, there are a couple of issues here:

1. it turns out that `freedesktop_os_release`, which we use for determining the actual name of the distro, is only available in Python 3.10 and later
2. assuming we have that method available, `ubuntu` is just a flavor of Linux, so we need at least one more arg like `flavor=None` to specify which Linux distro we are targeting
3. the `versions` arg is somewhat problematic since `freedesktop_os_release()['VERSION_ID']` actually returns something like `22.04`, so we either need to a) specify an exact match, or b) make a rather complex matching function. Since the tests mainly run in CI, which is always a predefined set (github actions always uses an LTS release of ubuntu), I'd prefer going with option a) to ease the maintenance burden.

Overall, given issue 1 above, I'm not sure what the right approach is. I _could_ write a custom replacement for it, but is it really worth it?